src/Session.php

```diff
@@ -118,9 +117,9 @@ public function getDriver()
      *
      * @return DocumentElement
      */
-    public function getPage()
+    public function getPage($xpath = '//html')
     {
-        return $this->page;
+        return $this->page = $this->page ?: new DocumentElement($xpath, $this);
```

Comment:
this logic is broken in case the page already exists and you pass a different XPath